src/current/releases/index.md

```diff
@@ -291,35 +314,35 @@ The following binaries are not suitable for production environments:
         </tr>
     </thead>
     <tbody>
-    {% for r in releases %}
+        {% for r in releases %}
```

Comment:
idk if you have an auto formatter, but I'd vote strongly against mixing in all these indentation changes along with the logic changes in this PR

Reply:
I re-indented them back to the correct level when I was trying to spot a Liquid nesting bug. The way indentation in this file is set up is complex and tends to drift toward incorrectness over time.